apps/site/src/middleware.page/return-types-as-json/hardcoded-types.ts

```diff
@@ -1,11 +1,12 @@
 /*
- @todo - We should be able to import from the `@blockprotocol/graph` package here but we're running into strange errors
-   with named exports, CommonJS, and ESM modules from the dependency on `/core`
+ @todo - We should be able to import these from the `@blockprotocol/graph` package here but we're running into strange errors
+   with "Dynamic Code Evaluation (e. g. 'eval', 'new Function', 'WebAssembly.compile') not allowed in Edge Runtime" only on
+   the integration tests CI job
  */
-import {
-  DATA_TYPE_META_SCHEMA,
-  ENTITY_TYPE_META_SCHEMA,
-} from "@blockprotocol/type-system/slim";
+const DATA_TYPE_META_SCHEMA =
+  "https://blockprotocol.org/types/modules/graph/0.3/schema/data-type";
+const ENTITY_TYPE_META_SCHEMA =
+  "https://blockprotocol.org/types/modules/graph/0.3/schema/entity-type";
```

Comment:
Even though the [previous commit fixed site building both locally and on Vercel](https://github.com/blockprotocol/blockprotocol/pull/1053/commits/629431153b0dd1ce4dd78cd149cff81619156c8e#r1118761439), [for some reason the _Site integration tests_ job couldn't build it](https://github.com/blockprotocol/blockprotocol/actions/runs/4282761095/jobs/7457881561#step:8:36)?!

```
Failed to compile.

../../libs/@blockprotocol/type-system/dist/es-slim/index-slim.js
Dynamic Code Evaluation (e. g. 'eval', 'new Function', 'WebAssembly.compile') not allowed in Edge Runtime 
Learn More: https://nextjs.org/docs/messages/edge-dynamic-code-evaluation

Import trace for requested module:
../../libs/@blockprotocol/type-system/dist/es-slim/index-slim.js
./src/middleware.page/return-types-as-json/hardcoded-types.ts
./src/middleware.page/return-types-as-json.ts
```